frontend/src/views/WheresMyGene/components/GeneSearchBar/components/QuickSelect/index.tsx

```diff
@@ -230,13 +243,11 @@ export default function QuickSelect<
           renderOption={renderOption}
           onPaste={handlePaste}
           InputBaseProps={{
-            onChange: (
-              event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
-            ) => {
-              setInput(event.target.value);
-            },
```

Comment:
The `onInputChange` on the Autocomplete component passes more args than the raw `Input`

Reply:
Ah I see why it's hard to type now. Apparently when we added `onInputChange` on SDS `MenuSelect`, we didn't realize it's colliding with MUI Autocomplete's `onInputChange`. Let's not use `onInputChange` until SDS works with CZ GenEpi on removing this unintended API collision 👌 

Thanks for exposing the issue!

Reply:
In that case, how should we address this bug?

Reply:
PR out! https://github.com/chanzuckerberg/sci-components/pull/156

Reply:
Do I need to introduce any changes to address the sds changes?

Reply:
We can upgrade `sds` to the latest and that should get the fix!